backend/tracim_backend/lib/utils/image_process.py

```diff
@@ -0,0 +1,57 @@
+from fractions import Fraction
+import typing
+
+from PIL import Image
+from PIL import ImageOps
+
+
+class ImageRatio:
+    def __init__(self, width: int, height: int):
+        self.width = width
+        self.height = height
+
+    def ratio(self) -> Fraction:
+        return Fraction(self.width, self.height)
+
+    def sized_to_ratio(self, image_size: "ImageSize") -> "ImageSize":
```

Comment:
wouldn't `crop` be clearer? I know we are manipulating ratios, but this would be unambiguous.

```suggestion
    def crop(self, image_size: "ImageSize") -> "ImageSize":
```

Reply:
This method doesn't crop, it just return the new size to fit to this ratio.